3.2/root/usr/bin/run-mongod-pet

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Run mongod as part of PetSet.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source ${CONTAINER_SCRIPTS_PATH}/common.sh
+
+function usage() {
+  echo "You must specify the following environment variables:"
+  echo "  MONGODB_USER"
+  echo "  MONGODB_PASSWORD"
+  echo "  MONGODB_DATABASE"
+  echo "  MONGODB_ADMIN_PASSWORD"
+  echo "  MONGODB_KEYFILE_VALUE"
+  echo "  MONGODB_REPLICA_NAME"
+  echo "Optional variables:"
+  echo "  MONGODB_SERVICE_NAME (default: mongodb)"
+  echo "MongoDB settings:"
+  echo "  MONGODB_NOPREALLOC (default: true)"
```

Comment:
Yes, I keep seeing in the logs that the config file contains values that do not apply.
AFAIK we haven't played much with that.